tests/test_sensors.py

```diff
@@ -111,11 +120,17 @@ def test_sensors(scene, has_sem, sensor_type, gpu2gpu, sim, make_cfg_settings):
     make_cfg_settings = {k: v for k, v in make_cfg_settings.items()}
     make_cfg_settings["semantic_sensor"] = has_sem
     make_cfg_settings["scene"] = scene
+    make_cfg_settings["frustum_culling"] = frustum_culling
 
     cfg = make_cfg(make_cfg_settings)
     for sensor_spec in cfg.agents[0].sensor_specifications:
         sensor_spec.gpu2gpu_transfer = gpu2gpu
 
+    # The scene loading may be done differently with/without frustum culling,
```

Comment:
Scene loading should be the same with/without FC, no special operations when it is on. :)
 If testing speed is not an issue, just keep it like this for precaution.

Reply:
The semantic mesh splitting is currently one done if frustum culling is enabled: https://github.com/facebookresearch/habitat-sim/blob/master/src/esp/sim/Simulator.cpp#L149